### DISCUSSION TITLE
sciter js for menu example ?

### DISCUSSION THREAD

**Discussion by toplinuxsir** [c-smile/sciter-js-sdk#101] [Q&A]
Where is menu example for sciter js ?

**Comment by MustafaHi** [comment]
https://github.com/c-smile/sciter-sdk/tree/master/samples/menu

**Comment by toplinuxsir** [comment]
> https://github.com/c-smile/sciter-sdk/tree/master/samples/menu
It is for tiscritp not for javascritp

**Comment by ArcRain** [comment]
A simple example is exist in the document [behavior-menu.md](https://github.com/c-smile/sciter-js-sdk/blob/main/docs/md/behaviors/behavior-menu.md)

**Comment by MustafaHi** [comment]
> > https://github.com/c-smile/sciter-sdk/tree/master/samples/menu
> > It is for tiscritp not for javascritp

It's using CSS so it works for both,
If by menu you mean popup then use `Element.popupAt` or `Element.popupFor`

https://github.com/c-smile/sciter-js-sdk/blob/main/docs/md/Element.md#popup
